litmus/director/openebs-install/test.yml

```diff
@@ -0,0 +1,171 @@
+        force_basic_auth: yes
+        return_content: yes
+        body_format: json
+        status_code: 200,202
+      register: node_cluster
+    
+    ## Getting node-1 id.
+    - name: get node-1 id
+      set_fact:
+        node1_id: "{{ node1_cluster.json.data[0].id}}"
+    
+    ## Getting node-2 id.
+    - name: get node-2 id
+      set_fact:
+        node2_id: "{{ node_cluster.json.data[1].id}}"
+
+    ## Labeling the node-1 of connected Cluster with controlPlaneNode=true and dataPlaneNode=true
+    - name: Giving POST request on labelnodes
+      uri:
+        url: "{{ director_url }}/v3/groups/{{ group_id }}/nodes/{{ node1_id }}/?action=labelnodes"
+        method: POST
+        url_username: "{{ username }}"
+        url_password: "{{ password }}"
+        force_basic_auth: yes
+        return_content: yes
+        body_format: json
+        body: '{"controlPlaneNode": true, "dataPlaneNode": true}'
+        status_code: 202
+
+    ## Labeling the node-2 of connected Cluster with controlPlaneNode=true and dataPlaneNode=flase
+    - name: Giving POST request on labelnodes
+      uri:
+        url: "{{ director_url }}/v3/groups/{{ group_id }}/nodes/{{ node2_id }}/?action=labelnodes"
+        method: POST
+        url_username: "{{ username }}"
+        url_password: "{{ password }}"
+        force_basic_auth: yes
+        return_content: yes
+        body_format: json
+        body: '{"controlPlaneNode": true, "dataPlaneNode": false}'
+        status_code: 202
+
+    ## Creating openebs
+    - name: Fetch openebses and create openebs
+      uri:
+        url: "{{ director_url }}/v3/groups/{{ group_id }}/openebses"
+        method: POST
+        url_username: "{{ username }}"
+        url_password: "{{ password }}"
+        force_basic_auth: yes
+        return_content: yes
+        body_format: json
+        body: "{ \"clusterId\": \"{{ cluster_id }}\",\"creatorId\": \"{{ group_id }}\",\"projectId\": \"{{ project_id }}\",\"templateId\": \"{{ template_id }}\",\"namespace\": \"{{ namespace }}\",\"defaultDirectory\": \"{{ default_directory }}\",\"dockerRegistry\": \"{{ docker_registry }}\",\"includeDeviceFilters\": \"{{ include_device_filters }}\",\"excludeDeviceFilters\": \"{{ exclude_device_filters }}\",\"cpuResourceLimit\": \"{{ cpu_resource_limit }}\",\"memoryResourceLimit\": \" {{ memory_resource_limit }}\",\"installationMode\": \"{{ installation_mode }}\" }"
+        status_code: 201
+      register: install_openebs
+
+    - name: Warning
+      debug:
+        msg: Openebs is Already  installed in your cluster.     
+      when: "{{ install_openebs.json.status }} == 405"    
+
+    - name: Getting the yaml for openebs installation
+      set_fact:
+        installopenebs: "{{ install_openebs.json.installationManifest }}"
+
+    - name: Getting the id
+      set_fact:
+        openebs_id: "{{ install_openebs.json.id }}"
+
+    - name: Getting into the openebs installation stage and checking the id
```

Comment:
Why is this task required?

Reply:
This task is to check the `openebs_id` which we are fetching in above task is correct or not